views/Data.tools.js

```diff
@@ -262,14 +270,13 @@ export function DataProvider(props) {
   return <Context.Provider value={value}>{props.children}</Context.Provider>
 }
 DataProvider.defaultProps = {
-  context: 'default',
```

Comment:
I think we need to leave `default` here (probably better) or use `let Context = DataContexts[props.context] || DataContexts.default` here https://github.com/viewstools/morph/pull/240/files#diff-4b21f1a692b6620ff5b97d354e20ef3a059e53e9cde024684401187964547a7eR91